Sources/MarathonCore/Run.swift

```diff
@@ -11,7 +11,7 @@ import Files
 
 public enum RunError {
     case missingPath
-    case failedToCompileScript([String])
+    case failedToCompileScript([String], String?)
```

Comment:
How about adding a label for the missing package, so it's a bit more clear what it refers to?